android/build.gradle

```diff
@@ -322,317 +156,22 @@ dependencies {
   implementation "androidx.camera:camera-extensions:1.1.0"
```

Comment:
all of these `androidx.camera:camera-*` libraries should be bumped to v1.2.3 (latest stable as of today: https://developer.android.com/jetpack/androidx/releases/camera#1.2.3)

![Screenshot 2023-08-01 at 2 18 11 PM](https://github.com/mrousavy/react-native-vision-camera/assets/4914611/b017d2df-094c-4105-ae3b-492e532d2e91)

in v2, there is some usage of the VideoCapture API that is causing a failure in session configuration somewhere in the codepath where the use cases are set

![Screenshot 2023-08-01 at 12 42 42 PM](https://github.com/mrousavy/react-native-vision-camera/assets/4914611/3d0297da-76bd-42ef-98cb-da1b60b6184e)

even if v3 doesn't use that exact same codepath anymore (not sure if it does), probably best to test with the latest libraries anyway

Reply:
Hey - V3 will not use CameraX anymore and will be rewritten to Camera2 :) In the middle of doing that